plugin/PDFView.qml

```diff
@@ -132,6 +136,35 @@ SilicaFlickable {
         linkColor: Theme.highlightColor;
         pagePlaceholderColor: Theme.highlightColor;
 
+        onPageLayoutChanged: if (rememberPositionConfig.value) restorePageSettings()
```

Comment:
Now I'm a bit puzzled for what actual case of pageLayoutChanged should restoring be applied to as it's emitted multiple times without order defined too well.

PageRectangle() is not needed if restoring uses only page number, not caring about zoom level. Both versions are fine, I guess.